stable/v3io-keycloak/values.yaml

```
@@ -0,0 +1,43 @@
+keycloak:
+  fullnameOverride: v3io-keycloak
+  postgresql:
+    enabled: false
+
+  extraEnv: |
+    - name: DB_VENDOR
+      value: mysql
+    - name: DB_ADDR
+      value: v3io-keycloak-db
+    - name: DB_PORT
+      value: "3306"
+    - name: DB_DATABASE
+      value: keycloak
+
+  extraEnvFrom: |
+    - secretRef:
+        name: 'v3io-keycloak-db-creds'
+
+  extraInitContainers: |
+    - name: wait-for-mysql
+      image: alpine:3.17
+      imagePullPolicy: IfNotPresent
+      command: ['sh', '-c']
+      args:
+        - echo -e "Waiting MySQL Server deployment availability";
+          while ! nc -z v3io-keycloak-db 3306; do
```

```suggestion
          while ! nc -z v3io-keycloak-db 3306; do
```
perhaps needs to be templated